app/src/main/java/com/codesoom/assignment/product/common/exception/ProductNotFoundException.java

```diff
@@ -0,0 +1,11 @@
+package com.codesoom.assignment.product.common.exception;
+
+/**
+ * 상품 ID에 해당하는 상품정보를 찾을수 없을때 발생하는 예외
+ */
```

Comment:
"발생"은 왜 일어났는지 모를 때 쓰는 단어입니다. 다른 사람이 만든 애플리케이션을 사용하는 입장이라면 쓸 수 있죠. 그러나 지금은 우리가 애플리케이션을 만들고 있는 입장입니다. 특정 상황에서 예외를 "던지도록" 직접 `throw`를 타이핑해두었죠.

```suggestion
/**
 * 상품정보를 찾을수 없을때 던집니다.
 */
```